onnxruntime/core/providers/webnn/builders/impl/squeeze_unsqueeze_op_builder.cc

```diff
@@ -80,46 +80,79 @@ Status UnsqueezeOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
     options.set("axes", emscripten::val::array(axes_data));
   }
 
-  emscripten::val output = model_builder.GetBuilder().call<emscripten::val>("unsqueeze", input, options);
+  emscripten::val output = emscripten::val::undefined();
+  if (node.OpType() == "Squeeze") {
+    output = model_builder.GetBuilder().call<emscripten::val>("squeeze", input, options);
+  } else {  // Unsqueeze
+    output = model_builder.GetBuilder().call<emscripten::val>("unsqueeze", input, options);
```

Comment:
I'd consistently mimic the pattern elsewhere, where you explicitly check for the name and then fail in the else. e.g.

https://github.com/microsoft/onnxruntime/blob/5eec24837ffa4d0519c242896f46a02d25c8e412/onnxruntime/core/providers/webnn/builders/impl/unary_op_builder.cc#L54-L57

```
...
  } else if (op_type == "Sin") {
    output = model_builder.GetBuilder().call<emscripten::val>("sin", input);
  } else if (op_type == "Sqrt") {
    output = model_builder.GetBuilder().call<emscripten::val>("sqrt", input);
  } else if (op_type == "Tan") {
    output = model_builder.GetBuilder().call<emscripten::val>("tan", input);
  } else {
    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT,
                           "UnaryOpBuilder::AddToModelBuilderImpl, unknown op: ", op_type);
  }
```

Reply:
OK